lib/wallaby.ex

```diff
@@ -32,6 +32,7 @@ defmodule Wallaby do
     children = [
       supervisor(Wallaby.Driver.ProcessWorkspace.ServerSupervisor, []),
       supervisor(driver(), [[name: Wallaby.Driver.Supervisor]]),
+      :hackney_pool.child_spec(:wallaby_pool, [timeout: 15_000, max_connections: 4]),
```

Comment:
I could not find docs on that in hackney, what is this timeout for? Is it a timeout for starting a worker or does it set default http request timeout for a worker?

Reply:
The timeout here is how long the connection will be kept alive in the pool.

Reply:
@keathley Doesn't it prevent the other configuration from being applied then? Sorry if asking stupid questions, never had to tweak hackney
```
config :wallaby,
  hackney_options: [timeout: :infinity, recv_timeout: :infinity]
```

Reply:
No worries :). Both of those options are for each individual request. The timeout option that is being added in this PR sets how long the TCP connection should be kept alive in-between requests.